modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -62,6 +62,7 @@ If you specify environment credentials, you can manage APIs of only that environ
 +
 For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster (only required for installation process)
+* If you have configured an external identity provider for your organization, create a user in Anypoint Platform with _Exchange Contributors_ and _Manage APIs Configuration_ roles. Anypoint Service Mesh does not support users residing on an external identity provider for tasks such as creating and discovering APIs.
```

Comment:
I was confused as "organization" typically means the Anypoint Platform instance. Can you add *where* (in what software or app) they've created the external IdP? (in the service they will add to service mesh? somewhere else?). I looked at the whole section but couldn't figure it out.

Reply:
Also, line 58 APIS should be APIs.

Reply:
Also, why are the permissions rendered in italics? If anything they should be in bold (if referring to the UI). You can skip this comment for now if the whole doc set is written that way, but we try to avoid italics if they aren't necessary.

Reply:
You can have master org and sub orgs. Each org can have its credentials reside on an external Idp, as required. Exteral Idp can be on any external Idp app. This a common practice and users would know. So I'd leave it as is for now. Thanks! Fixed the other two comments.